src/desktop/apps/personalize/client/onboarding.tsx

```diff
@@ -2,8 +2,8 @@ import React from "react"
 import ReactDOM from "react-dom"
 import { SystemContextProvider } from "v2/Artsy"
 import CollectorIntent from "v2/Components/Onboarding/Steps/CollectorIntent"
-import Artists from "v2/Components/Onboarding/Steps/Artists"
-import Genes from "v2/Components/Onboarding/Steps/Genes"
+import { ArtistsStep } from "v2/Components/Onboarding/ArtistsStep"
+import { GenesStep } from "v2/Components/Onboarding/GenesStep"
```

Comment:
👍

Reply:
So without index files would these lines look like this:

```ts
import { ArtistsStep } from "v2/Components/Onboarding/ArtistsStep/ArtistsStep"
import { GenesStep } from "v2/Components/Onboarding/GenesStep/GenesStep"
```

Or am I misunderstanding something? 🤔

Reply:
This is what we've been doing, assuming `ArtistsStep` and `GenesStep` need folders because they have related components; otherwise, things just just live flatly in `Onboarding`